README.md

```diff
@@ -28,30 +29,79 @@ ContribHub is a platform that connects developers with open-source projects that
 - TypeScript
 - Tailwind CSS
 - Supabase
-- GitHub API (Octokit)
+- GitHub API (via Octokit)
 
 ## 🚀 Getting Started
 
-A complete guide to setting up and running the project locally can be found in our [Contributing Guidelines](CONTRIBUTING.md).
+### Prerequisites
+
+Make sure you have the following tools installed:
+
+- Node.js
+- npm or yarn
+- Git
+- A Supabase account
+
+### Setting up Locally
+
+1. **Clone the repository**:
+   ```bash
+   git clone https://github.com/graphicmade/contribhub.git
+   cd contribhub
+   ```
+
+2. **Install dependencies**:
+   ```bash
+   npm install
+   ```
+   or
+   ```bash
+   yarn install
+   ```
+
+3. **Create a `.env` file** by copying the `.env.example` file and adding the necessary API keys and environment variables for Supabase and GitHub:
+   ```bash
+   cp .env.example .env
+   ```
+
+4. **Start the development server**:
+   ```bash
+   npm run dev
+   ```
+   The application should now be running on `http://localhost:3000`.
+
+### Importing the Database Schema to Supabase
+
+To import the schema from `seed.sql` to your local Supabase database, follow these steps:
+
+1. **Open the Supabase dashboard** and navigate to your project.
+2. Go to the **SQL** tab in the dashboard.
+3. **Copy the SQL content** from the `seed.sql` file located in the repository.
+4. **Paste the SQL** into the Supabase SQL editor and execute the query. This will set up the required database schema for ContribHub.
+5. After running the SQL script, your Supabase instance should be configured with the necessary tables and data.
+
+### Additional Configuration
+
+Once the schema is set up, make sure to update any remaining environment variables in your `.env` file to ensure the application connects properly to your Supabase instance.
```

Comment:
This is great, please put it in the [CONTRIBUTING.md](https://github.com/graphicmade/contribhub/blob/main/CONTRIBUTING.md) file

Reply:
you want the mentioned changes in CONTRIBUTING.md file instead of README.md file right?

Reply:
@swve review this :- I had moved the mentioned into CONTRIBUTING.md file from README.md file !

https://github.com/user-attachments/assets/23940586-36a8-456a-a254-a18fa2c9db56